mitodl/open-discussions
# Upgrade django

#### Pre-Flight checklist


#### What are the relevant tickets?
N/A

#### What's this PR do?
Upgrades django to the latest 2.x

#### How should this be manually tested?
Poke around the app. you should still be able to login, search, post things, edit your profile, upload images, etc.
